dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/utils/ContextUtils.java

```diff
@@ -189,6 +189,12 @@ private void configureResponse(
     }
   }
 
+  public static HttpHeaders getSecureNoCacheHeader() {
```

Comment:
I think using the word secure here is easily misleading the reader to think this has anything to do with security and authentication when it is just a working way of forcing the client to not cache the response. I also think we should maybe follow the spring methods in naming for this and simply call it `noCacheNoStoreMustRevalidate`.

Reply:
cool, i'll update